target/linux/ipq40xx/image/Makefile

```diff
@@ -257,6 +257,24 @@ define Device/engenius_ens620ext
 endef
 TARGET_DEVICES += engenius_ens620ext
 
+define Device/ezviz_cs-w3-wd1200g-eup
+	$(call Device/FitImage)
+	DEVICE_VENDOR := EZVIZ
+	DEVICE_MODEL := CS-W3-WD1200G
+	DEVICE_VARIANT := EUP
+	BOARD_NAME := cs-w3-wd1200g-eup
+	DEVICE_DTS_CONFIG := config@4
+	KERNEL_SIZE := 4096k
+	IMAGE_SIZE := 14848k
+	SOC := qcom-ipq4018
+	IMAGES := sysupgrade.bin
+	IMAGE/sysupgrade.bin := append-kernel | append-rootfs | pad-rootfs \
+		| append-metadata
```

Comment:
Keep the pipe in the first line (that seems to be the most frequent wrapping scheme used):

	IMAGE/sysupgrade.bin := append-kernel | append-rootfs | pad-rootfs | \
		append-metadata